### PR TITLE
Adding new Bill-related properties

### PR DESCRIPTION
**Pull request by lafritay**

### CODE REVIEW

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs [hunk 1]
```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
...
+        [JsonProperty("withdrawn_cosponsors")]
+        public int WithdrawnCosponsorCount { get; set; }
+
         [JsonProperty("primary_subject")]
         public string PrimarySubject { get; set; }
 
         [JsonProperty("committees")]
         public string Committees { get; set; }
 
+        [JsonProperty("committee_codes")]
```
Comment (babelshift) [inline ProPublicaCongressAPI/InternalModels/SpecificBill.cs:80]: I can't find this property in the documentation for the [Specific Bill Endpoint](https://projects.propublica.org/api-docs/congress-api/endpoints/#get-a-specific-bill). Where did you find it?

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs [hunk 2]
```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
...
+        public string[] CommitteeCodes { get; set; }
+
+        [JsonProperty("subcommittee_codes")]
```
Comment (babelshift) [inline ProPublicaCongressAPI/InternalModels/SpecificBill.cs:83]: I can't find this property in the documentation for the [Specific Bill Endpoint](https://projects.propublica.org/api-docs/congress-api/endpoints/#get-a-specific-bill). Where did you find it?

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs [hunk 3]
```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
...
+        [JsonProperty("active")]
+        public bool Active { get; set; }
+
+        [JsonProperty("house_passage")]
+        public string HousePassage { get; set; }
+
+        [JsonProperty("senate_passage")]
+        public string SenatePassage { get; set; }
+
+        [JsonProperty("enacted")]
```
Comment (babelshift) [inline ProPublicaCongressAPI/InternalModels/SpecificBill.cs]: Any idea if this is a true/false like the `active` property? The documentation just has blanks.

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs [hunk 4]
```diff
@@ -20,36 +26,84 @@ internal class SpecificBill
...
+        public string Enacted { get; set; }
+
+        [JsonProperty("vetoed")]
```
Comment (babelshift) [inline ProPublicaCongressAPI/InternalModels/SpecificBill.cs:65]: Any idea if this is a true/false like the `active` property? The documentation just has blanks.

**Review by babelshift — COMMENTED** [review]
Same comments for the Recent Bill changes as on the Specific Bill changes regarding some documentation discrepancies.

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs (hunk 4 above)
Comment (lafritay) [reply ProPublicaCongressAPI/InternalModels/SpecificBill.cs:65]: Oddly, it is a DateTime?.  I had to run a bunch of queries on the API to find examples.  It looks to be the same data as DateHousePassageVote and DateSenatePassageVote.  I'm wondering if they didn't like those names and decided to shorten them (both are documented).

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs (hunk 1 above)
Comment (lafritay) [reply ProPublicaCongressAPI/InternalModels/SpecificBill.cs:80]: I was going off of what I saw in fiddler:

image

Given they aren't documented, I'm happy to remove them.  Let me know what you'd like me to do there

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs (hunk 2 above)
Comment (lafritay) [reply ProPublicaCongressAPI/InternalModels/SpecificBill.cs:83]: Same as above.

**Comment by lafritay** [comment]
@babelshift Let me know if you'd prefer me to squash everything down to a single commit for submission... not sure what your preference is on history.

File: ProPublicaCongressAPI/InternalModels/SpecificBill.cs (hunk 1 above)
Comment (babelshift) [reply ProPublicaCongressAPI/InternalModels/SpecificBill.cs:80]: I think it's fine to add them if they're in the response. Maybe the documentation has yet to be updated.

**Comment by babelshift** [comment]
Thanks for the contribution.

**Comment by lafritay** [comment]
@babelshift Just out of curiosity, how often do you update the nuget feed?

**Comment by babelshift** [comment]
Sorry for the delayed response.

Normally the build on check in will create and deploy the Nuget package
automatically. After switching over to compiling with the newer VS2017
msbuild on AppVeyor, that process broke.

I will get it working in the next few days when I get some free time from
work.

**Comment by babelshift** [comment]
Auto Nuget deploys should be fixed now.

**Pull request merged**